test/yaml.test.ts

```
@@ -0,0 +1,908 @@
+        input = undefined;
+        result = AppOptions.resolveAddress(input, "SomeContractName", true);
+        assert.deepEqual(result, undefined);
+
+        // happy from env
+        process.env.INPUT = address;
+        input = "$INPUT";
+        result = AppOptions.resolveAddress(input, "SomeContractName");
+        assert.deepEqual(result, address);
+
+        // happy from env undefined
+        delete process.env.INPUT;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)